### DISCUSSION TITLE
Budgeted Column

### DISCUSSION THREAD

**Discussion by ColGra** [actualbudget/releases#119] [Feature ideas]
When clicking on the "Spent" column, that is the amount spent in any given category, one is presented with a very useful statement of all the transactions making up the balance. Would it not be a useful to have the same re the Budgeted column? YNAB (new) provides that information as does Goodbudget (to name two) and it is an ideal way to track transfers from category to category (including the "To Budget" category).

**Comment by Deguello** [comment]
Personally, I don't see any value at all in knowing how the money got budgeted into a category beyond, perhaps, Undo functionality... and that exists elsewhere.

I guess some people do, though. It doesn't look like that info is tracked at this point, but I could definitely be wrong about that. I haven't spent that much time looking deeply into the database.

**Reply by ColGra** [reply]
I experienced a huge issue in another piece off software because of that shortfall in info. Thankfully they eventually saw the light (for a change!). Money can be moved to/from incorrectly and that may not be noticed at the time. There is no audit trail to come back to and that flies in the face of standard accounting practice.

**Reply by Deguello** [reply]
Which might be bad if this were even remotely meant to conform to standard accounting practices, I suppose. It's not double entry bookkeeping, it's a budgeting app. I would hope it doesn't get too bogged down with juggling clowns and every possible "feature," but that's a decision for others to make.

**Reply by ColGra** [reply]
You are saying (or seem to be) that Actual is or should be no more that a glorified spreadsheet tracking spending against balances/targets that are of unknown construction. If that is its ultimate destination it will not be an acceptable option for some of the charity accounts I run. Those charities see each category as an account in its own right and moving money from "Peter to pay Paul" is a trackable event. I agree 100%. To have any financial package that does not conform to basic accounting principals is a huge mistake in my opinion, even if some ignore them. We shall have to see what the dev says.

**Reply by Deguello** [reply]
Not at all. I'm saying it is not, nor is it intended to be, as I understand it, a complete accounting package. Additional coding and overhead to try to make it mimic one is, in my opinion, a mistake. But, as I said, that's not my decision.

I'm not an accountant, but I was a lawyer, and I'd be of the opinion that combining multiple discrete charities into one budget, as well as commingling their funds in the same deposit accounts, is pretty likely to violate basic accounting principles, and fiduciary responsibilities, right from the outset so any software functionality designed to facilitate such a thing is (again), in my opinion, undesirable.

**Reply by ColGra** [reply]
If you were a lawyer you must know you are overstepping the mark. Where have I said the records/funds are co-mingled in any way whatsoever (or that that is my intention)? Where have I even said those charity budgets are in Actual? For what it is worth, the last thing I would do is place professional records in development software and development software with only a part time dev for support! As for your comment "any software functionality designed to facilitate such a thing is (again), in my opinion, undesirable" - you seem more focussed on poor legal opinions that basic accounting principals.

**Reply by Deguello** [reply]
These charities were your justification for why Actual needs to track budgeted amounts. If you don't want to have them in Actual, their existence, and your scenario, is irrelevant to Actual. If they were in Actual, in one budget, they'd be commingled.

There's no "if" I was a lawyer, but this is getting further off topic. If we're going to play the "where have I said" card, where did I say you did those things? I'm out.

**Reply by ColGra** [reply]
No - the charities were an example as to why the app is not necessarily satisfactory from an accounting perspective now. I could have used any example. You chose to turn that round and suggest that I might be breaching trust. Otherwise there was no reason for you to respond the way you did. Equally they would not be commingled if they were in Actual, not if they were in separate budgets. Again I never mentioned any form of structure so once again you are assuming things, seemingly making up your own little unfounded scenarios to suite your "opinions". Thank heavens you are now out.
